orchestrator-salt/src/main/resources/salt/salt/postgresql/scripts/create_embeddeddb_certificate.sh

```diff
@@ -0,0 +1,23 @@
+{%- set postgres_fqdn = salt['grains.get']('fqdn') %}
+{%- set cm_keytab = salt['pillar.get']('keytab:CM') %}
+#!/usr/bin/env bash
+set -e
+
+function cleanup() {
+  kdestroy
+}
+
+trap cleanup EXIT
+
+CM_KEYTAB_FILE={{ cm_keytab.path }}
+CM_PRINCIPAL={{ cm_keytab.principal }}
+CERTS_DIR=/opt/certs/postgres
+
+kinit -kt $CM_KEYTAB_FILE $CM_PRINCIPAL
+
+mkdir -p ${CERTS_DIR}
+
+openssl req -nodes -newkey rsa:2048 -days 365 -keyout ${CERTS_DIR}/postgres.key -out ${CERTS_DIR}/postgres.csr -subj "/CN={{postgres_fqdn}}"
+ipa cert-request ${CERTS_DIR}/postgres.csr --principal=postgres/{{ postgres_fqdn }} --add --certificate-out=${CERTS_DIR}/postgres.cert
```

Comment:
Perhaps `${CERTS_DIR}/postgres.csr` could be removed after a successful cert issuance.